src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+        echo "Prior probabilities file is provided as ${prior_prob}"
+        dx download "$prior_prob" -o inputs/prior_prob.tsv
+    else
+        mv prior_prob.tsv inputs/
+    fi
+
+    # Intervals file (preprocessed bed from GATK_prep) and annotation tsv (from GATK_prep)
+    dx download "$interval_list" -o inputs/beds/preprocessed.interval_list
+    dx download "$annotation_tsv" -o inputs/beds/annotated_intervals.tsv
+
+    SECONDS=0
+    echo "${bambais[@]}" | jq -r '.["$dnanexus_link"]' \
```

Shellcheck

[warning] bambais is referenced but not assigned.

(SC2154)